src/widgets/geo-search/geo-search.js

```diff
@@ -93,7 +93,6 @@ const suit = component('GeoSearch');
  *
  * Don't forget to explicitly set the `height` of the map container (default class `.ais-geo-search--map`), otherwise it won't be shown (it's a requirement of Google Maps).
  *
- * @type {WidgetFactory}
```

Comment:
API Extrator can't find this type, really don't know why. This file will be migrated to typescript anyways.